mdoc/src/main/scala/mdoc/internal/markdown/Processor.scala

```diff
@@ -171,7 +171,9 @@ class Processor(implicit ctx: Context) {
               inputFile,
               ctx.settings
             )
+            modifier.preProcess(postCtx)
             val postRender = modifier.process(postCtx)
```

Comment:
What's the purpose of `preProcess` and `postProcess` if the modifier could call those methods directly inside of `process`? Judging by the implementation, it doesn't look like mdoc does anything between `preProcess`, `process` and `postProcess`

```scala
class MyProcess extends PostProcess {
  def preProcess() = ()
  def process() = {
    preProcess()
    // ...
    postProcess()
  }
}
```

Reply:
You are right. This is incorrect. Does nothing. I think these calls should be removed. Ok with you? Want to change the `onStart` and `onExit` names ok?

Reply:
onStart and onExit are fine